### ISSUE TITLE
Strange border

### ISSUE THREAD

**Issue by Sean-Zeo** [smasherprog/screen_capture_lite#150]
Hi, when capturing a window frame on windows 10 there appears to be a small black border along the left and top of the image;
![image](https://user-images.githubusercontent.com/127526527/231138624-eef813f0-715e-4429-bac7-d46d8e7f56c8.png)

**Comment by smasherprog** [comment]
Ill check this out later today and get back with you.

**Comment by Sean-Zeo** [comment]
> Ill check this out later today and get back with you.

Hi friend, any update?

**Comment by smasherprog** [comment]
I cannot reproduce this problem. Can you try out the Example CPP project and run that code with these lines NOT commented out 
https://github.com/smasherprog/screen_capture_lite/blob/a76e6492761c01024334516e41d2594d62e2f0a5/Example_CPP/Screen_Capture_Example.cpp#L324

Then take a look at the output.  See if you can reproduce it using the example code.   You can comment out the other part of the test code, but run the code there and see if it still recurrs.

**Comment by Sean-Zeo** [comment]
Hi, here is the result;
![0WINNEW_](https://user-images.githubusercontent.com/127526527/232233611-ff18e598-114a-4b3a-9c60-83df678f8ccf.jpg)
![image](https://user-images.githubusercontent.com/127526527/232233620-70f313e9-fb27-4122-9835-83d0917575aa.png)

The black border still occurs.
-Sean

**Comment by Sean-Zeo** [comment]
Seems to be an offset of the buffer bytes of raw rgb?

**Comment by Sean-Zeo** [comment]
![image](https://user-images.githubusercontent.com/127526527/232351389-855d55af-7019-44ff-983f-062f7d7d44d1.png)
Temporary fix of the "illegal byte sequence" error is to only allow letters, numbers, spaces & special characters.

**Comment by smasherprog** [comment]
this  fixes the issue?

**Comment by Sean-Zeo** [comment]
> this fixes the issue?

No, this only fixes the "illegal byte" crash when converting the w.Name to a string via; `[MarshalAs(UnmanagedType.ByValTStr, SizeConst = 128)]`

This is unrelated to the border issue found when grabbing window frames.

**Comment by smasherprog** [comment]
Since i cant reproduce the problem i cant really help much unfortunately. :(

**Comment by smasherprog** [comment]
I can take a look at that illegal character issue. Looks like your visiting github issues page while capturing a window. I can check that out.

**Comment by Sean-Zeo** [comment]
> Since i cant reproduce the problem i cant really help much unfortunately. :(

Hi why can't you reproduce? Do you not have access to a windows 10 machine?
-Sean

**Comment by smasherprog** [comment]
I have windows 10 and windows 11 and it doesnt occur on either machines capturing a window. There is no black border around it :(

**Comment by smasherprog** [comment]
Its going to be something different on your machine than mine, so once that difference is identified we can fix it. Theres always little things that cause issues like this, but finding them is very hard

**Comment by Sean-Zeo** [comment]
Hi, just testing on a virtual machine, vps && a different laptop, the issue still occurs?
Can you show an example of it running correctly?  The only window that seems to work without a border is task manager.
Thanks!

**Comment by smasherprog** [comment]
You have put alot of effort into this and i appreciate it.  I am heading into work not but  when i get home from work ill get back at it and try some things out.

**Comment by smasherprog** [comment]
This is super anooooying.. Its in  https://github.com/smasherprog/screen_capture_lite/blob/cb777c9f84c47866612272c1486f21abfd4e5953/include/windows/GDIHelpers.h#L26

Ive been messing with it, but havent been able to fix it.   TAG ur it!
